oracle-linux-image-tools/bin/build-image.sh

```diff
@@ -2,7 +2,7 @@
 #
 # Create minimal Oracle Linux images
 #
-# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 1982-2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Was this script really originally created in 1982? If not, this copyright line is incorrect. Also, we no longer include "All rights reserved." in our copyright lines in 2020, so that bit can be removed as well.

Reply:
Updated -- bf5ca17